test/nostale_crypto/native_test.exs

```diff
@@ -46,4 +46,20 @@ defmodule NostaleCrypto.NativeTest do
 
     assert NostaleCrypto.Native.login_decrypt(enc) == dec
   end
+
+  test "world_next/2 return split by delimiter" do
+    enc =
+      <<198, 228, 203, 145, 70, 205, 214, 220, 208, 217, 208, 196, 7, 212, 73, 255, 208, 203, 222,
+        209, 215, 208, 210, 218, 193, 112, 67, 220, 208, 210, 63, 199, 228, 203, 161, 16, 72, 215,
+        214, 221, 200, 214, 200, 214, 248, 193, 160, 65, 218, 193, 224, 66, 241, 205, 199, 228,
+        203, 161, 16, 72, 215, 214, 221, 200, 214, 200, 214, 248, 193, 160, 65, 218, 193, 224, 66,
+        241, 205>>
+
+    assert NostaleCrypto.Native.world_next(enc, 0) ==
+             {<<198, 228, 203, 145, 70, 205, 214, 220, 208, 217, 208, 196, 7, 212, 73>>,
+              <<208, 203, 222, 209, 215, 208, 210, 218, 193, 112, 67, 220, 208, 210, 63, 199, 228,
+                203, 161, 16, 72, 215, 214, 221, 200, 214, 200, 214, 248, 193, 160, 65, 218, 193,
+                224, 66, 241, 205, 199, 228, 203, 161, 16, 72, 215, 214, 221, 200, 214, 200, 214,
+                248, 193, 160, 65, 218, 193, 224, 66, 241, 205>>}
+  end
```

Comment:
- Add a test with a packet that doesn't not contains the delimiter 
- Add a test with a "big" number as key like `40000`